build.gradle

```diff
@@ -101,6 +101,12 @@ dependencies {
 
 sourceCompatibility = JavaVersion.VERSION_1_7
 
+signing {
+  if (hasProperty('SIGNING_PRIVATE_KEY')) {
+    useInMemoryPgpKeys(SIGNING_PRIVATE_KEY, "")
```

Comment:
Is this a Gradle API?

Reply:
Yes, I found that through sqldelight 
https://docs.gradle.org/current/userguide/signing_plugin.html#sec:signatory_credentials